bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -304,6 +316,69 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
     {% endif -%}
     {% endfor -%}
 
+    {% for table_sensor_task in task.depends_on_tables_existing -%}
+    {% set project_id, dataset_name, table_name = table_sensor_task.table_id.split('.') -%}
+    {{ table_sensor_task.task_id }} = BigQueryTableExistenceSensor(
+        task_id={{ table_sensor_task.task_id | format_repr }},
+        project_id={{ project_id | format_repr }},
+        dataset_id={{ dataset_name | format_repr }},
+        table_id={{ table_name | format_repr }},
+        gcp_conn_id='google_cloud_shared_prod',
+        deferrable=True,
+        {% if table_sensor_task.poke_interval != None -%}
+        poke_interval={{ table_sensor_task.poke_interval | format_timedelta | format_repr }},
+        {% else -%}
+        poke_interval=datetime.timedelta(minutes=5),
+        {% endif -%}
+        {% if table_sensor_task.timeout != None -%}
+        timeout={{ table_sensor_task.timeout | format_timedelta | format_repr }},
+        {% else -%}
+        timeout=datetime.timedelta(hours=8),
```

Comment:
I chose 8 hours as the default timeout so that when combined with the common DAG-level defaults of 2 retries it would result in the sensor waiting for ~24 hours total before giving up and failing entirely.